app/javascript/components/patient/close_contacts/CloseContactTable.js

```diff
@@ -0,0 +1,489 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Card, Col, Dropdown, Form, InputGroup, OverlayTrigger, Row, Tooltip } from 'react-bootstrap';
+
+import { formatDate } from '../../../utils/DateTime';
+import { formatPhoneNumber } from '../../../utils/Patient';
+
+import axios from 'axios';
+import _ from 'lodash';
+
+import InfoTooltip from '../../util/InfoTooltip';
+import CloseContactModal from './CloseContactModal';
+import confirmDialog from '../../util/ConfirmDialog';
+import CustomTable from '../../layout/CustomTable';
+import reportError from '../../util/ReportError';
+import DeleteDialog from '../../util/DeleteDialog';
+
+class CloseContactTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      table: {
+        colData: [
+          { label: 'Actions', field: '', isSortable: false, className: 'text-center', filter: this.renderActionsDropdown },
+          { label: 'First Name', field: 'first_name', isSortable: true },
+          { label: 'Last Name', field: 'last_name', isSortable: true },
+          { label: 'Phone Number', field: 'primary_telephone', isSortable: true, filter: formatPhoneNumber },
+          { label: 'Email', field: 'email', isSortable: true },
+          { label: 'Last Date of Exposure', field: 'last_date_of_exposure', isSortable: true, filter: formatDate },
+          { label: 'Assigned User', field: 'assigned_user', isSortable: true },
+          { label: 'Contact Attempts', field: 'contact_attempts', isSortable: true, filter: v => (v.value ? v.value : '0') },
+          { label: 'Enrolled?', field: 'enrolled_id', isSortable: true, filter: v => (v.value ? 'Yes' : 'No') },
+          // Lodash's `truncate` includes an ellipses, and will only fire when the length is greater than `length`
+          // However, it throws a false-positive for our linter (which thinks we're calling fs.truncate)
+          /* eslint-disable security/detect-non-literal-fs-filename */
+          { label: 'Notes', field: 'notes', isSortable: true, filter: v => _.truncate(v.value, { length: 503 }) },
+        ],
+        rowData: [],
+        totalRows: 0,
+        selectedRows: [],
+        selectAll: false,
+      },
+      query: {
+        page: 0,
+        entries: 10,
+      },
+      entryOptions: [10, 15, 25],
+      cancelToken: axios.CancelToken.source(),
+      isLoading: false,
+      editRow: null,
+      showEditModal: false,
+      showAddCloseContactModal: false,
+      showDeleteModal: false,
+      selectedCloseContactID: null,
```

Comment:
can we camel case Id here?

Reply:
can we actually combine this with `editRow` and call it `activeRow`? they're essentially doing the same thing

Reply:
Really good catch here